source/archive/organizing-conversations.rst

```diff
@@ -75,5 +75,5 @@ For additional details about working with the Beta release of Collapsed Reply Th
 .. raw:: html
 
    <div style="position: relative; padding-bottom: 50%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
-      <iframe src="https://www.youtube.com/embed/hnD0Zj-mIbs" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
+      <iframe src="https://www.youtube.com/embed/hnD0Zj-mIbs" alt="video on organizing converstion" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```

Comment:
@shikhar13012001 - Source files located in the ``archive`` folder are no longer built into the product documentation and shouldn't be updated. Please revert this change.